test/cmd/lfstest-gitserver.go

```diff
@@ -1116,6 +1133,49 @@ func skipIfBadAuth(w http.ResponseWriter, r *http.Request, id string) bool {
 	return true
 }
 
+func handleNTLM(w http.ResponseWriter, r *http.Request, authHeader string, session ntlm.ServerSession) {
+	if strings.HasPrefix(strings.ToUpper(authHeader), "BASIC ") {
+		authHeader = ""
+	}
+
+	switch authHeader {
+	case "":
+		w.Header().Set("Www-Authenticate", "ntlm")
+		w.WriteHeader(401)
+
+	// ntlmNegotiateMessage from httputil pkg
+	case "NTLM TlRMTVNTUAABAAAAB7IIogwADAAzAAAACwALACgAAAAKAAAoAAAAD1dJTExISS1NQUlOTk9SVEhBTUVSSUNB":
```

Comment:
It could, but it's already in `httputil` (and will be moved to `lfsapi`. Could just export it there, and use here.